test/built-ins/Object/values/exception-during-enumeration.js

```diff
@@ -2,8 +2,8 @@
 // This code is governed by the BSD license found in the LICENSE file.
 
 /*---
+id: sec-object.values
```

Comment:
Same as `Object.entries`, we should wait until the proposal gets merged into the draft to use the ID.